### ISSUE TITLE
wrong dates!

### ISSUE THREAD

**Issue by SepidehAlassi** [dasch-swiss/beol#75]
Date presentations in BEOL are incorrect for some of the resources. For resource which has   

  "knora-api:dateValueHasCalendar": "GREGORIAN",
    "knora-api:dateValueHasEndEra": "CE",
    "knora-api:dateValueHasEndYear": 1786,
    "knora-api:dateValueHasStartEra": "CE",
    "knora-api:dateValueHasStartYear": 1786,
    "knora-api:hasPermissions": "CR knora-base:Creator|M knora-base:ProjectMember|V knora-base:KnownUser|RV knora-base:UnknownUser",
    "knora-api:valueAsString": "GREGORIAN:1786 CE"
Date is correct in triplestore and knora returns correct date value. 
But in GUI the shown date is 1785 CE!!

<img width="746" alt="screenshot 2018-11-28 16 43 46" src="https://user-images.githubusercontent.com/8567642/49167208-da37ca00-f32c-11e8-8ca2-c8b82ccc55f3.png">

In the new data please check the resource 
http:%2F%2Frdfh.ch%2F0801%2F0TRvVX9kSyabvaD5J1_7tQ

Since the problem is not true for all resources this sounds really fishy!

**Comment by tobiasschweizer** [comment]
his could be related to the native JS date object which is used for display. I will check that, too.

**Comment by tobiasschweizer** [comment]
maybe it is because months indexes are 0 based. And there is no month here: -1 => November of the previous year.

**Comment by SepidehAlassi** [comment]
yeah it seems like it is rounding with -1 to previous year! But what puzzles me is that it does not do the same thing for all dates which have just year precision!!!!!

**Comment by tobiasschweizer** [comment]
does it depend on the browser? we had some problems with Safari.

**Comment by SepidehAlassi** [comment]
I am using chrome

**Comment by SepidehAlassi** [comment]
@tobiasschweizer It is browser based!!! in Safari date is right!!!!!

![screenshot 2018-11-28 17 09 48](https://user-images.githubusercontent.com/8567642/49168886-6ac3d980-f330-11e8-8daf-08d45a57e5bb.png)

**Comment by tobiasschweizer** [comment]
Could you please make an issue in Knora-UI and assign it to @flavens, referring to this issue? 

I bet it is a problem with the constructor of the date object.

**Comment by tobiasschweizer** [comment]
the component in questions is: kui-date-value (viewer module)

**Comment by tobiasschweizer** [comment]
@andreas-aeschlimann suspsects that it could also be a time zone related problem.

**Comment by andreas-aeschlimann** [comment]
Commented in the other thread @tobiasschweizer.
